include/sway/desktop/fx_renderer.h

```diff
@@ -11,6 +11,8 @@ struct decoration_data {
 	float alpha;
 	float saturation;
 	int corner_radius;
+	float dim;
+	float* dim_color;
```

Comment:
float dim_color[4];

Reply:
The thing is that I won't be able to directly assign it. I could use a `memcpy` though:
``` c
memcpy(deco_data.dim_color, view_is_urgent(focus->view)
	? config->dim_inactive_colors.urgent
	: config->dim_inactive_colors.unfocused, sizeof(float[4]));
```

Is there another way that I'm not familiar with? I'm definitely not a C expert 😅

Reply:
Ah you know what, you're probably right, as is would be best